includes/assets/stories-widget.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+(function ($) {
+  /**
+   * Store field states in local variable.
+   *
+   * @type {Object}
+   */
+  var fieldState = window.webStoriesData || {};
+
+  /**
+   * Closure for making inputs reactive
+   * on view type selection change.
+   *
+   * @param {Object}  target Event object.
+   * @param {boolean} reset Flag to reset fields value.
+   */
+  var react = function (target, reset = false) {
+    var currentView = target.value;
+    var widget = target.closest('.widget');
+    var state = fieldState.fields[currentView];
+
+    for (var [key, value] of Object.entries(state)) {
+      var field = widget.querySelector('.' + key + '.stories-widget-field');
+      var fieldWrapper = widget.querySelector('.' + key + '_wrapper');
```

Comment:
Should we use jquery for selection here. It is already used in other places.

Reply:
Don't think we need to use jQuery for such simple selectors, but I'd use template literal strings for improved readability.

Although... why are we using jQuery here in the first place?

Reply:
This is my question, either use jQuery or es6 selectors. Using a mix of the both seems weird. I think es6 is better, but we could use jQuery as everything else is.

Reply:
Removed couple of jQuery selectors. Currently, only the jQuery event listeners are there.